eng/common/pipelines/templates/steps/credscan.yml

```diff
@@ -4,15 +4,17 @@ parameters:
   SourceDirectory: $(Build.SourcesDirectory)
   ServiceDirectory: ''
 
+# cSpell:ignore changedfiles
```

Comment:
Does this work? also is the multiple cases of credscan needed?

Reply:
It is working. I have suppressed all complains. https://dev.azure.com/azure-sdk/public/_build/results?buildId=1441957&view=logs&jobId=b70e5e73-bbb6-5567-0939-8415943fadb9&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=bdeefc16-b669-5ebd-ad94-a2c19ade53b0


I did not check if it is case insensitive. I can remove one for testing.